app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetsController < Sinatra::Base
+
+  include Helper
+
+  configure do
+    set :public_folder, 'public'
+    set :views, "app/views/tweets"
+    enable :sessions
+    set :session_secret, "what a wonderful secret this is"
+    use Rack::Flash
+  end
+
+  get "/tweets" do
+    @user = current_user
+    @tweets = Tweet.all
+    erb :tweets
+  end
+
+  get "/tweets/new" do
+    erb :new
+  end
+
+  post "/tweets" do
+    if params[:tweet][:content].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.